isotovideo

```diff
@@ -410,16 +409,17 @@ if ($testfd) {
     kill_autotest;
 }
 
-diag "isotovideo done" unless $r;
-diag "FAIL" if $r;
+diag "isotovideo " . ($r ? 'failed' : 'done');
```

Comment:
we don't know the job state at this point (and it doesn't matter) - succeeded expresses more than we know